spec/lucky/action_spec.cr

```diff
@@ -224,6 +224,22 @@ describe Lucky::Action do
     end
   end
 
+  describe ".permitted_query_params" do
```

Comment:
How about query_param_declarations? Permitted query params makes me think of operations and that it’ll return the params somehow. I think param declarations might be a bit clearer

Thoughts?

Reply:
I struggled with this name lol. Yeah, that sounds good to me 👍